src/Common/ForceAsyncAwaiter.cs

```diff
@@ -34,7 +34,8 @@ internal struct ForceAsyncAwaiter : ICriticalNotifyCompletion
 
         public ForceAsyncAwaiter GetAwaiter() { return this; }
 
-        public bool IsCompleted { get { return false; } } // the purpose of this type is to always force a continuation
+        public bool IsCompleted // the purpose of this type is to always force a continuation
```

Comment:
Fix this formatting so the comment is above the property and the => false is on the same line